bot/handlers/user/subscription/payments_stars.py

```
@@ -0,0 +1,148 @@
+    i18n_data: dict,
+    session: AsyncSession,
+    stars_service: StarsService,
+):
+    payload = (message.successful_payment.invoice_payload
+               if message and message.successful_payment else "")
+    try:
+        parts = (payload or "").split(":")
+        payment_db_id = int(parts[0])
+        months = float(parts[1]) if len(parts) > 1 else 0
+        sale_mode = parts[2] if len(parts) > 2 else "subscription"
+    except Exception:
```

Ruff (0.15.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)